introduction/07-faq.asciidoc

```diff
@@ -0,0 +1,15 @@
+== FAQ
+
+To conclude the Introduction segment of the Learning Path, here are some Frequently Asked Questions people have when embarking on their InnerSource journey.
+
+=== What is the cost/overhead?
+
+=== Why not just open source?
+
+=== Will this slow us down?
+
+=== Will we just be reviewing PRs all day every day?
+
+=== How do we convince management this is a good idea?
```

Comment:
Figure out what they want and get a working example of InnerSource that shows them getting it.

Reply:
> working example of InnerSource
I assume you mean from the wider industry? Blogs, conference talks etc.?

Reply:
Ah yeah that would definitely be an easier sell than pointing at some other company that might have very different culture and ways of working.

Reply:
https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/465/commits/0e4a7d0a88dfae20331af1a86b47e3b1ba80eef2

Reply:
Two recommendations:
- Add link: https://innersourcecommons.org/stories/
- If your organization's OSPO manages innersource projects, reach out to them for support.